### PR TITLE
[CHK 2369] add new closure requested event

### PR DESCRIPTION
**Pull request by Fernando-Granato**
#### List of Changes

1. Added handling of a new **v2** event. The new event is the `CLOSURE_REQUESTED_EVENT` useful for handling asynchronous close payment. 
2. The changes have lead to touch the flow of events.
     

- From the state `AUTHORIZATION_COMPLETED` we will now move to either `CLOSURE_REQUESTED` or `EXPIRED` 
-  From the `CLOSURE_REQUESTED` state we will be able to move to the `CLOSURE_ERROR` `CLOSED` `CLOSURE_FAILED` and  `EXPIRED`

4. Also added a new state for the `CLOSURE_REQUESTED` view.

#### Motivation and Context
The change was made to handle asynchronous close payment only for v2 events

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: specs/state_machine.js [hunk 1]
```diff
@@ -17,6 +17,17 @@ createMachine(
         },
       },
       AUTHORIZATION_COMPLETED: {
+        entry: assign({ auth_outcome: "OK" }),
```
Comment (infantesimone) [inline specs/state_machine.js:20]: given that the sequence diagram changes between v1 and v2, maybe it's better to split the two diagrams?

**Review by infantesimone — APPROVED** [review]
sounds good

File: specs/state_machine.js (hunk 1 above)
Comment (Fernando-Granato) [reply specs/state_machine.js:20]: Good idea 👍🏻

Comment (Fernando-Granato) [reply specs/state_machine.js]: Fixed with c6e8c869fa14c39ce89afcad9a06027241ca4e60 🚀

**Review by infantesimone — APPROVED** [review]

**Pull request merged**